kubetest2-gke/deployer/build/gke_make.go

```diff
@@ -112,6 +116,23 @@ func (gmb *GKEMake) Stage(version string) error {
 	if err := gmb.runWithActions(os.Stdout, os.Stderr, []gkeBuildAction{stage}, args...); err != nil {
 		return err
 	}
+
+	if gmb.UpdateLatest {
+		r := regexp.MustCompile(gkeMinorVersionRegex)
+		m := r.FindStringSubmatch(version)
+		if len(m) < 2 {
+			return fmt.Errorf("failed to update the latest version because can't find the minor version of %s", version)
+		}
+		minor := m[1]
+		fName := fmt.Sprintf("latest-%s.txt", minor)
+		pushCmd := fmt.Sprintf("echo '%s' | gsutil cp - %s/%s", version, gmb.StageLocation, fName)
```

Comment:
You can set the stdin for the cmd, to avoid echo and bash.

Reply:
Done